Update FAQ and Landpage (#2791)
naturalcrit/homebrewery: client/homebrew/pages/homePage/welcome_msg.md

\column

## V3 vs Legacy
The Homebrewery has two renderers: Legacy and V3. The V3 renderer is recommended and will receive updates, while the Legacy renderer will not. Legacy mode is still available for older brews and veteran users, but won't be updated.
:

> V3 uses curly brackets instead of HTML tags, though HTML tags can still be used. Converting between Legacy and V3 may require tweaking the document. However, all legacy brews will still render normally. Both renderers can be accessed via the {{fa,fa-info-circle}} **Properties** button on your brew at any time.

Author: I like this revision, but think this still refers to V3 as something brand new and different.
It is my opinion that we would benefit of using a different terminology, as V3 has been the default and therefore the "normal" for some years now, and there's been talk about V4.

\column

## V3 vs Legacy

> The Homebrewery has two renderers: Legacy and V3. The V3 renderer is recommended and will receive updates, while the Legacy renderer will not. Legacy mode is still available for older brews and veteran users, but won't be updated.
> :
> V3 uses curly brackets instead of HTML tags, though HTML tags can still be used. Converting between Legacy and V3 may require tweaking the document. However, all legacy brews will still render normally. Both renderers can be accessed via the {{fa,fa-info-circle}} **Properties** button on your brew at any time.

Reviewer: ```suggestion
The Homebrewery has two renderers: Legacy and V3. The V3 renderer is recommended for all users because it is more powerful, more customizable, and continues to receive new feature updates while Legacy does not. However Legacy mode will remain available for older brews and veteran users.
	
At any time, any individual brew can be changed to your renderer of choice via the {{fa,fa-info-circle}} **Properties** tab on your brew. However, converting between Legacy and V3 may require heavily tweaking the document; while both renderers can use raw HTML, V3 prefers a streamlined curly bracket syntax that avoids the complex HTML structures required by Legacy.
```

Scroll down to the next page for a brief summary of the changes and features available in V3!
#### New Things All The Time!
Check out the latest updates in the full changelog [here](/changelog).

### Helping out

> Like this tool? Want to buy us a beer? [Head here](https://www.patreon.com/Naturalcrit) to help us keep the servers running.

Reviewer: ```suggestion
Like this tool? Head over to our [Patreon](https://www.patreon.com/Naturalcrit) to help us keep the servers running.

```

If you are looking for more 5e Homebrew resources check out [r/UnearthedArcana](https://www.reddit.com/r/UnearthedArcana/) and their list of useful resources [here](https://www.reddit.com/r/UnearthedArcana/wiki/resources). The [Discord Of Many Things](https://discord.gg/by3deKx) is another great resource to connect with fellow homebrewers for help and feedback.

{{position:absolute;top:20px;right:20px;width:auto
[![Discord](/assets/discord.png){height:30px}](https://discord.gg/by3deKx)
[![Github](/assets/github.png){height:30px}](https://github.com/naturalcrit/homebrewery)
[![Patreon](/assets/patreon.png){height:30px}](https://patreon.com/NaturalCrit)
[![Reddit](/assets/reddit.png){height:30px}](https://www.reddit.com/r/homebrewery/)
}}

\page

## Markdown+
The Homebrewery aims to make homebrewing as simple as possible, providing a live editor with Markdown syntax that is more human-readable and faster to write with than raw HTML.

From version 3.0.0, with a goal of adding maximum flexibility without users resorting to complex HTML to accomplish simple tasks, Homebrewery provides an extended verision of Markdown with additional syntax.

### Curly Brackets

> Markdown is a simplified version of HTML, as such there are equivalences like the replacement of `<span></span>` and `<div></div>` with `{{ }}` for a cleaner custom formatting. Inline spans and block elements can be created and given ID's and Classes, as well as css properties, each of which are comma separated with no spaces. Use double quotes if a value requires spaces. Spans and Blocks start the same:

Reviewer: ```suggestion
Standard Markdown lacks several equivalences to HTML. Hence, we have introduced `{{ }}` as a replacement for `<span></span>` and `<div></div>` for a cleaner custom formatting. Inline spans and block elements can be created and given ID's and Classes, as well as CSS properties, each of which are comma separated with no spaces. Use double quotes if a value requires spaces. Spans and Blocks start the same:
```
